ote/resources/public/language/en.edn

```diff
@@ -664,7 +664,7 @@
   :basic-info-available-to "Valid to"
 
   :route-calendar-name "Reittikalenteri"
-  :route-calendar-group-name "Driving days calendar"
+  :route-calendar-group-name ["Service calendar (Departs from " :departure-stop " at " :departure-time ")"]
```

Comment:
In Finnis we have "Liikennöinti", in Swedish "trafiken" but in english "service". Should it be "Traffic" of "route"?